functests/cluster_upgrade_test.go

```diff
@@ -46,7 +47,7 @@ var _ = Describe("Cluster upgrade", func() {
 				By("Getting the current csv before the upgrade")
 				csv, err := deployManager.GetCsv()
 				Expect(err).To(BeNil())
-				
```

Comment:
Don't introduce unrelated (even whitespace) changes if we can avoid it.

Reply:
It's a linter fix. There was initially few "Tabs" which is replaced by NewLine character.

Reply:
hm ok. that could/should have been a separate commit as well. :-)